node/antieframework.js

```diff
@@ -113,38 +112,22 @@ var AntieFramework = function(configPath, frameworkPath) {
      * @param string pageStrategy The page strategy used by this device.
      * @param string element The page strategy property to return (Sub-directory of {{_configPath}}/config/pagestrategy/{{pageStrategy}}/{{element}}
      * directory).
-     * @param string default The default value to return if the page strategy does not contain the requested element.
      * @return string An element (property) of the page strategy or the default value.
      */
     var getPageStrategyElement = function(pageStrategy, element) {
```

Comment:
That is the eventual plan, but it isn't ready to be open sourced yet. This is only intended to be a step on the way to that goal. Getting rid of this bootstrap file entirely would leave people worse off than they are now - it would require them to put configs and page strategies together themselves. It would also require us to rewrite the simple Node server in the talexample project, which nobody wants to do (well, I don't).